examples/grpc-service/src/main/java/example/armeria/grpc/Main.java

```diff
@@ -0,0 +1,52 @@
+package example.armeria.grpc;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.linecorp.armeria.common.grpc.GrpcSerializationFormats;
+import com.linecorp.armeria.server.Server;
+import com.linecorp.armeria.server.ServerBuilder;
+import com.linecorp.armeria.server.docs.DocService;
+import com.linecorp.armeria.server.grpc.GrpcServiceBuilder;
+
+import io.grpc.protobuf.services.ProtoReflectionService;
+
+public final class Main {
+
+    private static final Logger logger = LoggerFactory.getLogger(Main.class);
+
+    public static void main(String[] args) throws Exception {
+        final Server server = newServer(8080, 8443);
+
+        Runtime.getRuntime().addShutdownHook(new Thread(() -> {
+            server.stop().join();
+            logger.info("Server has been stopped.");
+        }));
+
+        server.start().join();
+        logger.info("Server has been started.");
```

Comment:
How about printing the URL to the docservice page, so that a user can play with it quickly? (Maybe we should do the same for other examples with docservice.)